R/execute.R

```diff
@@ -110,10 +110,12 @@ execute_array_udf <- function(namespace, array, udf, layout=NULL, selectedRanges
   # object.
   if (typeof(resultObject) != "raw") {
     className <- class(resultObject)[1]
+    print(resultObject)
+    print(str(resultObject))
     if (className == "ApiResponse") {
-      stop(paste("tiledbcloud: received error response:", resultObject$content))
+      stop(paste("tiledbcloud: received error response:", resultObject$content, sep="\n"))
```

Comment:
Not sure if that was initially generated that way but `paste()` inside `stop()` is redundant as `stop()` already glues arguments together---try `stop("AB", "CD", "DE", call. = FALSE)`.

I am not sure the `\n` is a good idea either.  But that gets to tastes.

There is a handy base R function `strwrap(txt, width)` that reformats `txt` to `width`.  But then this an error printer, not a pretty printer.

Reply:
Thanks @eddelbuettel !!

There are lots of API callsites ... I think I'll take the bits I wrote, put them in a helper function, then have the various API callsites call it on non-2xx cases ... then flip this PR from RFC to non-RFC

Thanks for the tips on `stop`/`paste`/etc! :) Your R expertise continues to be wonderful to draw from :)